include/openenclave/internal/sgxsign.h

```diff
@@ -25,6 +25,8 @@ OE_EXTERNC_BEGIN
  * @param security_version[in] ISVSVN value for the SGX sigstruct
  * @param pem_data[in] PEM buffer containing the signing key
  * @param pem_size[in] size of the PEM buffer
+ * @param family_id[in] ISVFAMILYID value for the SGX sigstruct
+ * @param ext_product_id[in] ISVEXTPRODID value for the SGX sigstruct
```

Comment:
Several "ext_product_id" need to be renamed

Reply:
Renamed